### ISSUE TITLE
Investigate effective bundle size for understanding sink options

### ISSUE THREAD

**Issue by jklukas** [mozilla/gcp-ingestion#501]
We'd like to understand how big our bundles tend to be for telemetry streaming jobs reading from PubSub. If they are large enough, we may be able to write a custom GCS sink that is hella more efficient than the built-in FileIO.Write and BigQueryIO.Write.

**Comment by jklukas** [comment]
Probably easiest way of doing this would be to add a simple ParDo that keeps track of number of records seen and then emits size in the `@FinalizeBundle` method.

**Comment by jklukas** [comment]
I did a test deploy of #514 on a subscription to the full telemetry topic, so full rate of messages coming in like production. 4 instances and using all defaults. I disabled everything in the job other than reading from PubSub, decompressing, and publishing these bundle metrics. No sinks to complicate things or potentially slow down consumption. Let it run for two hours.

Screenshot of metrics below:

<img width="464" alt="Screen Shot 2019-04-04 at 1 21 06 PM" src="https://user-images.githubusercontent.com/691060/55581392-b9bcb980-56ea-11e9-9fa2-6c4e89f1d12a.png">

Mean of ~500 elements per batch and mean of ~10 MB uncompressed data per batch. My tentative conclusion here is that batches are generally going to be too small for us to write out an object to GCS per batch.
